packages/components/src/alignment-matrix-control/README.md

```diff
@@ -5,14 +5,61 @@ AlignmentMatrixControl components enable adjustments to horizontal and vertical
 ## Usage
 
 ```jsx
-import { AlignmentMatrixControl } from '@wordpress/components';
+import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
 import { useState } from '@wordpress/element';
 
 const Example = () => {
-	const [ alignment, setAlignment ] = useState( 'center center' );
+	const [alignment, setAlignment] = useState('center center');
 
 	return (
-		<AlignmentMatrixControl value={ alignment } onChange={ setAlignment } />
+		<AlignmentMatrixControl
+			value={alignment}
+			onChange={(newAlignment) => setAlignment(newAlignment)}
+		/>
 	);
 };
```

Comment:
Very very minor, but — it'd be great if the example snippet followed the repo's [prettier formatting rules](https://github.com/WordPress/gutenberg/blob/375eaa0d870282e12cd461a53463d3e10e776c41/packages/prettier-config/lib/index.js)

```suggestion
import { __experimentalAlignmentMatrixControl as AlignmentMatrixControl } from '@wordpress/components';
import { useState } from '@wordpress/element';

const Example = () => {
	const [ alignment, setAlignment ] = useState( 'center center' );
	return (
		<AlignmentMatrixControl
			value={ alignment }
			onChange={ ( newAlignment ) => setAlignment( newAlignment ) }
		/>
	);
};
```